delfin/drivers/dell_emc/vmax/perf_utils.py

```diff
@@ -0,0 +1,108 @@
+# Copyright 2020 The SODA Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import time
+
+# minimum interval supported by VMAX
+VMAX_PERF_MIN_INTERVAL = 5
```

Comment:
Move these to constants file

Reply:
moving to new constants file in VMAX, as these are VMAX specific constants.

Reply:
ok